wayland-client/src/event_queue.rs

```diff
@@ -730,3 +730,32 @@ macro_rules! delegate_dispatch {
         )*
     };
 }
+
+/// A helper macro which delegates a set of [`Dispatch`] implementations for proxies to a no-op handler.
+///
+/// # Usage
+///
+/// This macro is useful to implement [`Dispatch`] for interfaces where events are unimportant to
+/// the current application and can be ignored.
+///
+/// ```
+/// use wayland_client::{delegate_noop, protocol::wl_subcompositor};
+/// delegate_noop!(ExampleApp: [wl_subcompositor::WlSubcompositor]);
+/// ```
+#[macro_export]
+macro_rules! delegate_noop {
```

Comment:
This macro should probably be relocated to `dispatch.rs`

Reply:
There's no `dispatch.rs` for `wayland_client`, only for `wayland_server`. `delegate_dispatch` is also in this file so this seemed like the best fit.